WalletWasabi.Fluent/ViewModels/AddWallet/RecoverWalletViewModel.cs

```diff
@@ -75,6 +78,14 @@ public partial class RecoverWalletViewModel : RoutableViewModel
 				});
 
 			EnableAutoBusyOn(NextCommand);
+
+			DeleteSuggestionCommand = ReactiveCommand.Create<object>((del) =>
```

Comment:
fairly sure viewmodel doesnt need to be involved at all with this.

Reply:
This is like a textbox having a delete command, so when the viewmodel detects a backspace... it calling the delete command and the viewmodel has to remove a character... to update the control.